src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -173,12 +173,14 @@ private static void MutationFunctionsTestSetup(RecalcEngine engine, bool numberI
              * Record rwr2 => {![Field1:n, Field2:![Field2_1:n, Field2_2:s, Field2_3:![Field2_3_1:n, Field2_3_2:s]], Field3:b]}
              * Record rwr3 => {![Field1:n, Field2:![Field2_1:n, Field2_2:s, Field2_3:![Field2_3_1:n, Field2_3_2:s]], Field3:b]}
              * Record r_empty => {}
+             * Record r3 => ![a:*[b:n]]
              * Table t1(r1) => Type (Field1, Field2, Field3, Field4)
              * Table t2(rwr1, rwr2, rwr3)
              * Table t_empty: *[Value:n] = []
              * Table t_empty2: *[Value:n] = []
              * Table t_an_bs: *[a:n,b:s] = []
              * Table t_name: *[name:s] = []
+             * Table t_bs: *[b:s] 

```

Comment:
>* Table t_bs: *[b:s]  [](http://example.com/codeflow?start=13&length=21)

The comment says that we're adding `t_bs`, but the code adds `t_bs1` and `t_bs2`

Reply:
fixed